Update documentation about System32 directory in FIM (#5765)
wazuh/wazuh-documentation: source/user-manual/capabilities/file-integrity/fim-configuration.rst

    <windows_registry arch="32bit" check_all="no" check_mtime="yes">HKEY_LOCAL_MACHINE\SYSTEM\Setup</windows_registry>
  </syscheck>

Configuring Windows directories
-------------------------------

In 64-bit architecture systems, you can locate 32-bit and 64-bit DLLs in a special way.

-  ``System32`` is reserved for 64-bit DLLs.
-  ``SysWOW64`` is reserved for all 32-bit DLLs.

Furthermore, 32-bit processes running in 64-bit environments access ``Sytem32`` through a virtual folder called ``Sysnative``.

> But this redirection has been disabled and you can access ``System32`` directly. Monitoring ``%WINDIR%/System32`` and ``%WINDIR%/Sysnative`` directories is equivalent and Wazuh shows the path ``%WINDIR%/System32`` in the alerts. ``SysWOW64`` is a different directory. To monitor ``%WINDIR%/SysWOW64``, you must add it to the ``ossec.conf`` configuration file.

Reviewer: ```suggestion
But we disabled this redirection and you can access ``System32`` directly. Monitoring ``%WINDIR%/System32`` and ``%WINDIR%/Sysnative`` directories is equivalent and Wazuh shows the path ``%WINDIR%/System32`` in the alerts. ``SysWOW64`` is a different directory. To monitor ``%WINDIR%/SysWOW64``, you must add it to the ``ossec.conf`` configuration file.
```

  </syscheck>

.. _how_to_fim_windows_directories:

Configuring Windows directories
-------------------------------

> To configure directories within the Windows directory (Also known as ``%WINDIR%``), it is necessary to take into account that in 64-bit architecture systems, there are two special folders: ``System32`` which is reserved for those 64-bit DLLs, and ``SysWOW64`` for all 32-bit DLLs.

Reviewer 2: ```suggestion
To configure Windows special directory ``%WINDIR%\system32``, it is necessary to take into account that in 64-bit architecture systems: 
1) **System32** is reserved for those 64-bit DLLs.
2) **SysWOW64** for all 32-bit DLLs.

```

The 32-bit processes running in a 64-bit environment, when they want to access Sytem32, they do it through a virtual folder called ``Sysnative``.

> From version 4.4.0 of Wazuh, this redirection has been disabled and it is possible to access directly to ``System32``, so those configurations that have as monitoring directory ``%WINDIR%/Sysnative`` from now on will generate alerts, synchronizations with the database and other functions with ``%WINDIR%/System32``. That means, ``%WINDIR%/Sysnative`` or ``%WINDIR%/System32`` represent the same directory and both path are valids.

Reviewer 2: ```suggestion
From version 4.4.0 of Wazuh, this redirection has been disabled and it is possible to access directly to ``System32``, so monitoring directories ``%WINDIR%/System32`` or ``%WINDIR%/Sysnative`` will be equivalent, and the path shown in the alerts will be ``%WINDIR%/System32``.
```

  </syscheck>

.. _how_to_fim_windows_directories:

Configuring Windows directories
-------------------------------

To configure Windows special directory ``%WINDIR%\system32``, it is necessary to take into account that in 64-bit architecture systems: 
1) **System32** is reserved for those 64-bit DLLs.
2) **SysWOW64** for all 32-bit DLLs.

The 32-bit processes running in a 64-bit environment, when they want to access Sytem32, they do it through a virtual folder called ``Sysnative``.

> From version 4.4.0 of Wazuh, this redirection has been disabled and it is possible to access directly to ``System32``, so monitoring directories ``%WINDIR%/System32`` or ``%WINDIR%/Sysnative`` will be equivalent, and the path shown in the alerts will be ``%WINDIR%/System32``.

Author: Done!
